invenio_app_rdm/config.py

```diff
@@ -725,6 +725,13 @@ def files_rest_permission_factory(obj, action):
         "content-type": "application/ld+json",
         "filename": "{id}.json",
     },
+    "csv": {
```

Comment:
Is this still needed? Otherwise, we can close this PR (or move it to non-draft if it's needed)

Reply:
Only if we want to display the export CSV button in the UI. I've left it as a draft as I think we are going just the API route and can be closed if you agree